loader:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint5_velocity_controller:
    type: effort_controllers/JointEffortController
    joint: front_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint1_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}  
  joint3_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint4_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel4_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
